Sources/ScipioKit/Producer/PIF/BuildParametersGenerator.swift

```diff
@@ -92,6 +106,25 @@ struct BuildParametersGenerator {
         return filePath
     }
 
+    func generate(from buildOptions: BuildOptions, toolchain: UserToolchain) async -> BuildParameters {
+        let arch = try? await executor.execute([
```

Comment:
In SwiftPM, the target architecture is obtained from `swiftc -print-target-info`, but with this approach we have to parse the triple string like `"triple": "arm64-apple-macosx15.0"`,  so we use arch instead.

ref: 
https://github.com/swiftlang/swift-package-manager/blob/5da13be6bd815e5c7323288e00c8b203542bc48d/Sources/SPMBuildCore/BuildParameters/BuildParameters.swift#L172

https://github.com/swiftlang/swift-package-manager/blob/5da13be6bd815e5c7323288e00c8b203542bc48d/Sources/Basics/Triple%2BBasics.swift#L92